src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -74,11 +78,46 @@
   static final String ENCRYPTED_FILE_EXTENSION = ".enc";
   static final String MISSING_FILE_EXTENSION = ".missing";
 
+  static final File cacheFile = new File(getFormsFolder().getParent(), "cache.ser");
+
+  static Map pathToHashMap;
+  static Map pathToDefinitionMap;
+
   public static final String getMountPoint() {
     return System.getProperty("os.name").startsWith("Win") ? File.separator + ".." : (System
         .getProperty("os.name").startsWith("Mac") ? "/Volumes/" : "/mnt/");
   }
 
+  public static void saveFormDefinitionCache() {
+    if (!cacheFile.exists() || cacheFile.canWrite()) {
```

Comment:
This should create the definition cache file if it's not exists, or if it's exists then only try to write to it if it's writable. I think the writable part is not needed?